README.md

```diff
@@ -49,22 +49,25 @@
     - [Strato](#strato)
     - [LoopiaSE](#loopiase)
     - [Infomaniak](#infomaniak)
+    - [Hetzner](#hetzner)
     - [OVH](#ovh)
     - [Dynu](#dynu)
+    - [IONOS](#ionos)
   - [Notifications](#notifications)
     - [Email](#email)
     - [Telegram](#telegram)
     - [Slack](#slack)
     - [Discord](#discord)
     - [Pushover](#pushover)
   - [Webhook](#webhook)
-    - [HTTP GET Request](#webhook-with-http-get-reqeust)
-    - [HTTP POST Request](#webhook-with-http-post-request)
+    - [Webhook with HTTP GET reqeust](#webhook-with-http-get-reqeust)
+    - [Webhook with HTTP POST request](#webhook-with-http-post-request)
   - [Miscellaneous topics](#miscellaneous-topics)
     - [IPv6 support](#ipv6-support)
     - [Network interface IP address](#network-interface-ip-address)
     - [SOCKS5 proxy support](#socks5-proxy-support)
     - [Display debug info](#display-debug-info)
+    - [Multiple API URLs](#multiple-api-urls)
```

Comment:
There are some changes here, that I didn't make manually, so I'm not 100% sure about those.